src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+  
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+  
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+  
+.search-form {
+  position: relative;
+  margin-top: 20px;
+}
+  
+.filed-input::placeholder {
+  font-size: 16px;
+  font-family: inherit;
+  color: #3d4e61;
+}
+  
+.filed-input {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

Comment:
use [inherit](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit#:~:text=The%20inherit%20CSS%20keyword%20causes,needed%20to%20override%20another%20rule.)